tests/containers/podman_netavark.pm

```diff
@@ -49,6 +49,12 @@ sub is_cni_default {
 }
 
 sub remove_subtest_setup {
+    if ($dev) {
+        script_run 'ip a s';
```

Comment:
While in a manual check is nice to simplify typing, in a automated scripting I'd suggest to make clear the command and write explicitly the full parameters:
`ip address show`

Reply:
Yeah, true that we should use the longer options. This is so ubiquitous in our code base, I guess it does not matter in a cleanup routine and in a command that is surely well known.

Reply:
That's networking people's bad habit Maurizio :) Instead `show running-config interface GigabitEthernet 0/15` you just type `sh ru int Gi 0 15` :smile: